src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,11 @@
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.hidden {
```

Comment:
probably you don't need this class `.hidden`

Reply:
Deleted